spaceship/lib/spaceship/tunes/tunes_client.rb

```diff
@@ -811,6 +817,83 @@ def release!(app_id, version)
     end
 
     #####################################################
+    # @!group purchases
+    #####################################################
+    
+    def purchases(app_id: nil)
+      r = request(:get, "ra/apps/#{app_id}/iaps")
+      return r.body["data"]
+    end
+    
+    def delete_purchase!(app_id: nil, purchase_id: nil)
+      r = request(:delete, "ra/apps/#{app_id}/iaps/#{purchase_id}")
+      handle_itc_response(r)
+    end
+    
+    def create_purchase!(app_id: nil,type: nil,  versions: nil, reference_name: nil, product_id: nil, cleared_for_sale: true, review_notes: nil, review_screenshot: nil, tier: nil)
```

Comment:
It would be great to have comments above each method with a short description of the method and each parameter